src/styles/main.css

```
@@ -67,6 +67,64 @@ p.error-msg {
   height: 100%;
 }
 
+.application-form-wrapper {
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+  font-family: Andale Mono, monospace;
+}
+
+.application-form {
+  background-color:	#f0f0f0
+}
+
+.application-form-wrapper h1 {
+  font-weight: 600;
+}
+
+.application-form-wrapper h5 {
+  font-weight: 600;
+  color: #910000;
+}
+
+.application-form-wrapper h6 {
+  font-weight: 600;
+  color: #910000;
+}
+
+.application-form-wrapper p {
+  font-size: 12px;
+}
+
+.application-form-wrapper .thanks-text p {
+  font-size: 20px;
+}
+
+.application-form-wrapper .row {
+	padding-top: 50px;
+}
+
+.application-form-wrapper .row-button {
+	padding-top: 50px;
+	padding-bottom: 50px;
+	text-align: center;
```

Nit: more indentation

```suggestion
  padding-top: 50px;
  padding-bottom: 50px;
  text-align: center;
```